sdk/core/core/inc/az_result.h

```diff
@@ -17,8 +17,9 @@
 
 enum {
   AZ_FACILITY_CORE = 0x1,
-  AZ_FACILITY_JSON = 0x2,
-  AZ_FACILITY_HTTP = 0x3,
+  AZ_FACILITY_PLATFORM = 0x2,
+  AZ_FACILITY_JSON = 0x3,
+  AZ_FACILITY_HTTP = 0x4,
```

Comment:
IOT and MQTT facilities should be here; did they get lost in a merge?

Reply:
I saw their PR, it is not yet checked in, I think they will merge now